src/main/__tests__/apiClient.spec.ts

```diff
@@ -117,10 +117,10 @@ describe('loginFromSession', () => {
         toQueryString({
           client_id: clientId,
           response_type: 'code',
-          scope: 'openid profile email phone',
-          display: 'page',
           redirect_uri: redirectUri,
           id_token_hint: idTokenHint,
+          scope: 'openid profile email phone',
```

Comment:
Because of some changes in `computeAuthParam`, the order of params in the QS has changed. So this is sadly a required change.

I think the underlying issue being that we test against a string and not a list of parameters instead.